Enterprise-CMCS/macpro-onemac
# Show 2nd clock with Pending statuses (sans Pending - RAI)

Story: https://qmacbis.atlassian.net/browse/OY2-24621

### Details

This PR adds a `span` that says 2nd Clock under the status when a package has a Pending, Pending - Concurrence, or Pending - Approval status **&&** the `latestRaiResponseTimestamp` is not null/undefined.

### Changes

- Add condition and span
- Update `one-seed` for testing data

### Implementation Notes

- N/A

### Test Plan

1. Log in as a CMS User
2. Navigate to the Dashboard and optionally filter the dashboard by filter to find all Pendgin, Pending - Concurrence, and Pending - Approval packages with RAI responses
3. Confirm:
     - [MD-22-2203-VM (localhost link)](http://localhost:3000/detail/medicaid-spa/MD-22-2203-VM): Pending - Approval and RAI received Feb 9 '23 -- shows 2nd Clock under status on details page
     - [MD-22-2204-VM (localhost link)](http://localhost:3000/detail/medicaid-spa/MD-22-2204-VM): Pending - Concurrence and RAI received Feb 9 '23 -- shows 2nd Clock under status on details page
     - [MD-23-3333-VM (localhost link)](http://localhost:3000/detail/medicaid-spa/MD-23-3333-VM): Pending and RAI received Feb 9 '23 -- shows 2nd Clock under status on details page
     - [MD-22-2305-VM (localhost link)](http://localhost:3000/detail/chip-spa/MD-22-2305-VM): Pending and NO RAI received -- DOES NOT show 2nd Clock
